_sass/_md.scss

```diff
@@ -1,4 +1,20 @@
 @media #{$md} {
+	.searchfield {
+		top: 82px;
+        width: 85vw;
```

Comment:
I think i can remove the "vw" setting now. I needed this before I found the class="container-float" option (the search field was not resizing to match the screen width and extended off the screen).